install-android-sdk.sh

```diff
@@ -0,0 +1,15 @@
+#/usr/bin/env bash
```

Comment:
Can you please move this file to the "it-android" directory so that the root directory doesn't get too full?

Could you also make it executable (might need to mess around with Git -- usually if you "chmod a+x" the file Git will figure it out when you first add it).